### ISSUE TITLE
Intermediate changes (e.g. gamera changes) need to be documented 

### ISSUE THREAD

**Issue by sabrina0822** [DDMAL/Rodan#1020]
There are some intermediate changes that we have made, such as our own modifications to our forked gamera repo, that need to be documented for future reference. 

@malajvan @softcat477 the gamera changes and tfaip changes come to mind to me personally when I think about random stuff we changed. Is there anything else you guys can think of?

**Comment by malajvan** [comment]
Yeah, I can speak for gamera itself and a section in the wiki and add documentation in the gamera fork readme would be good. 
I'll look through DDMAL github to see other forks we might be using although cant think of any on top of my head

**Comment by sabrina0822** [comment]
Ok thanks @malajvan, I think you know more about gamera than I do so I'll let you document those changes.

**Comment by malajvan** [comment]
Gamera changes are added. See #1044
